tests/components/sensor/test_rest.py

```diff
@@ -24,13 +25,6 @@ def tearDown(self):
         """Stop everything that was started."""
         self.hass.stop()
 
-    def test_setup_missing_config(self):
```

Comment:
Instead of removing this test, you can check that the no platforms are loaded under the sensor component, using:
```py
with assert_setup_component(0):
    assert setup_component(self.hass, rest.DOMAIN, {'sensor': {'platform': 'rest'}})
```
See https://github.com/home-assistant/home-assistant/blob/dev/tests/common.py#L393